sklearn/metrics/_scorer.py

```diff
@@ -476,15 +482,15 @@ def check_scoring(estimator, scoring=None, *, allow_none=False):
     scoring : callable
         A scorer callable object / function with signature
         ``scorer(estimator, X, y)``.
+
+    Notes
+    -----
+    For evaluating multiple scores, use
+    :func:`~sklearn.model_selection.cross_validate` instead.
```

Comment:
I don't think we need this note. `cross_validate` is different from `check_scoring` which returns a scorer.